docs/product-assembly_designing-and-building-cases/src/main/asciidoc/case-management-rules-con.adoc

```diff
@@ -0,0 +1,7 @@
+[id='case-management-rules-con']
+= Rules in case management 
+
+Cases are data-driven, rather than following a clear sequential flow. The steps required to resolve a case rely on data, which is provided by people involved in the case, or the system can be configured to trigger further actions based on the data available. In the latter case, business rules can be used to decide what further actions are required for the case to continue or reach a resolution.
```

Comment:
Cases are data-driven, rather than following a clear sequential flow.

Might want to remove "clear". Just a thought.